File: PROGRAMAS/3_MAXIMO_COMUN_DIVISOR/maximo_comun_divisor.m
1;
function u = Maximo_comun_divisor(grande,peque)
	residuo = mod(grande,peque);
	while (residuo !=0)
		grande = peque;
		peque  = residuo;
		residuo = mod(grande,peque);
	end
	u = peque;
end

format long
disp(Maximo_comun_divisor(6983776800,5587021440))

File: PROGRAMAS/3_MAXIMO_COMUN_DIVISOR/maximo_comun_divisor.py
def maximo_comun_divisor(grande,peque):
    residuo = grande % peque
    while residuo !=0:
        grande = peque
        peque = residuo
        residuo = grande % peque
    return peque

print(maximo_comun_divisor(6983776800,5587021440))
